releases/release-1.35/exceptions.yaml

```
@@ -0,0 +1,148 @@
+# Exception requests in v1.35
+# Google Group: https://groups.google.com/g/kubernetes-sig-release
+# Release Team Lead: Drew Hagen ([@drewhagen](https://github.com/drewhagen) )
+
+# Enhancements Freeze Exceptions requested in v1.35
+
+enhancementFreeze:
+- name: "Mutable CSINode Allocatable Property"
+  issue: 4876
+  date_requested: 2025-10-17
+  date_reviewed: 2025-10-19
```

Based on the Approval email date converted to UTC.

```suggestion
  date_reviewed: 2025-10-18
```